UI/broadcast-flow.cpp

```diff
@@ -0,0 +1,116 @@
+// SPDX-FileCopyrightText: 2023 tytan652 <tytan652@tytanium.xyz>
+//
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "broadcast-flow.hpp"
+
+void OBSBroadcastFlow::ManageBroadcast(bool streamingActive)
+{
+	if (flow.manage_broadcast2) {
```

Comment:
Check the new structure in the OBS Frontend API and it's related documentation.

TLDR: There is two variant of the `manage_broadcast`.

Reply:
I didn't check if the a certain flag is set 🤦 while checking for the `2` variant.

Reply:
Got it - I dunno what other maintainers think about this, but in general I really don't like the current practise of just slapping a number to an existing function when we extend its signature - it's a quick and dirty workaround, but not good API design.

Given that we are creating a new API from scratch here, we can be more elegant and this case I'd rather give the function a more descriptive name, because so far "2" means "new variant, use this", but these 2 functions can and should be used depending on functionality of the service.

So I'd suggest using something closer to `manage_broadcast_with_stream_status` or something similar. ObjC and Swift both make extensive use of identical functions with different signatures (allowing different variations of parameters) and verbose function names that tell you _by themselves_ what they do.